genesyscloud/outbound_contact_list_contact/resource_genesyscloud_outbound_contact_list_contact_utils.go

```diff
@@ -181,3 +181,15 @@ func GenerateColumnStatus(column, contactable string) string {
 			contactable = %s
 		}`, column, contactable)
 }
+
+func createComplexContact(contactListId, contactId string) string {
+	return fmt.Sprintf("%s:%s", contactListId, contactId)
+}
+
+func splitComplexContact(complexContact string) (string, string) {
+	if strings.Contains(complexContact, ":") {
```

Comment:
add delimiter as a constant, in the package  so that test cases also would refer the same everywhere.